src/creature_control.h

```diff
@@ -141,8 +141,8 @@ struct CreatureControl {
     unsigned char flgfield_2;
     unsigned char combat_flags;
     unsigned char party_objective;
-    long field_5;
-    short field_9;
+    long delay;
```

Comment:
could be `unsigned long` because it is a turn time actually.
Maybe call it `wandering_time` ?

Reply:
I'm using it for this fear delay now too.